marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/artifacts/core.sjs

```diff
@@ -249,7 +249,7 @@ function deleteArtifactSettings(artifactType, artifactName, artifactVersion = 'l
     return { success: true };
 }
 
-function getEntityNames() {
+function getEntityTitles() {
```

Comment:
This is fine here - just an FYI, we'll soon have a library for functions pertaining to entity models, and this will be refactored to use that library.